.sauce/sentry-uitest-android-ui.yml

```diff
@@ -3,15 +3,15 @@ kind: espresso
 sauce:
   region: us-west-1
   # Controls how many suites are executed at the same time (sauce test env only).
-  concurrency: 1
+  concurrency: 4
```

Comment:
I think I've read somewhere that `concurrency` has no effect on the real devices, it's only relevant when running on emulators, but I might be wrong.

Reply:
the only reference to real devices vs emulator for concurrency i found is -> https://support.saucelabs.com/hc/en-us/articles/225243308-Why-am-I-not-getting-the-parallelism-concurrency-I-expected-
It just says: `Real Devices have a separate concurrency limit.`
While here it talks about concurrency on real devices -> https://docs.saucelabs.com/mobile-apps/faq/#test-queuing-and-concurrency